src/main/java/com/nike/cerberus/command/core/InitializeEnvironmentCommand.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.command.core;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.beust.jcommander.ParametersDelegate;
+import com.nike.cerberus.command.Command;
+import com.nike.cerberus.domain.cloudformation.TagParametersDelegate;
+import com.nike.cerberus.operation.Operation;
+import com.nike.cerberus.operation.core.InitializeEnvironmentOperation;
+
+import java.util.List;
+
+import static com.nike.cerberus.command.core.InitializeEnvironmentCommand.COMMAND_NAME;
+
+/**
+ * Command for creating the base components for Cerberus.
+ */
+@Parameters(commandNames = COMMAND_NAME,
+        commandDescription = "Create the IAM roles, KMS keys, and S3 buckets and base config files for needed to " +
+                "bootstrap a Cerberus environment and store env infrastructure state")
+public class InitializeEnvironmentCommand implements Command {
+
+    public static final String COMMAND_NAME = "initialize-environment";
```

Comment:
nitpick: Could be shortened to "init-env" or "init-environment".

Reply:
sure sounds good to me